### PR TITLE
Update NGFF converted studies

### PR DESCRIPTION
**Pull request by dominikl**
That's the stats for current prod122 release. If someone (@sbesson ?) could pop in the correct db size into the releases.tsv please. I can't ssh into prod122 omero to get it.

### CODE REVIEW

File: _data/releases.tsv [hunk 1]
```diff
@@ -75,3 +75,4 @@ Date	Data release	Code version	Sets	Wells	Experiments	Images	Planes	Size (TB)	Fi
 2023-10-09	prod119	0.11.9	21617	1978974		14012392	118390729	384.893134429384	38.160129	468.92	125
 2024-03-05	prod120	0.12.0	21628	1978974		14017840	118660875	385.67365086156303	38.176475	471.31	127
 2024-05-20	prod121	0.12.1	21660	1978974		14018967	118814421	399.030210159843	38.179857	471.45	128
+2024-07-03	prod122	0.12.2	21660	1978974		14018967	118814421	409.427319995011	200.818337	123	128
```
Comment (sbesson) [inline _data/releases.tsv:78]:
```suggestion
2024-07-03	prod122	0.12.2	21660	1978974		14018967	118814421	409.427319995011	482.32	123	128
```

**Review by sbesson — APPROVED** [review]
Database size updated with the state of `prod122`. Leaving @will-moore and @francesw to approve these changes and we can update the stats.

**Review by will-moore — APPROVED** [review]
Looks good. I assume the method for generating the stats for NGFF s3 data is documented and checked elsewhere?

**Comment by sbesson** [comment]
> Looks good. I assume the method for generating the stats for NGFF s3 data is documented and checked elsewhere?

Slowly going over emails, I suspect the relevant script is in https://github.com/IDR/idr-utils/pull/65 and I have assigned you as a reviewer.

**Pull request merged**